buildpack.md

```diff
@@ -658,31 +658,31 @@ If the environment variable file name ends in `.prepend`, then the value of the
 In either case, within that environment variable value,
 - Later buildpacks' environment variable file contents MUST precede earlier buildpacks' environment variable file contents.
 - Environment variable file contents originating from the same buildpack MUST be sorted alphabetically descending by associated layer name.
-- Environment variable file contents originating in the same layer MUST be sorted such that file contents in `<layers>/<layer>/env.build/` or `<layers>/<layer>/env.launch/` precede file contents in `<layers>/<layer>/env/`.
+- **Environment variable file contents originating in the same layer MUST be sorted such that file contents in `<layers>/<layer>/env.launch/<process>/` precede file contents in `<layers>/<layer>/env.launch/`, which must precede `<layers>/<layer>/env/`.**
```

Comment:
```suggestion
- **Environment variable file contents originating in the same layer MUST be sorted such that file contents in `<layers>/<layer>/env.launch/<process>/` precede file contents in `<layers>/<layer>/env.build` or `<layers>/<layer>/env.launch/`, which must precede `<layers>/<layer>/env/`.**
```

Did you intentionally leave out `env.build`?

Reply:
this is copied directly from the RFC, but yes I think this is correct. The `env.build` vars aren't honored at launch, so there's no concept of "process specific" build vars

Reply:
eh, i think we still need it in some way

Reply:
yup, you were right

Reply:
I actually think it might be confusing to list both build and launch handling in the same sentence. They are entirely different processes. The build order will never have any process specific files . The previous sentence only dealt with the one difference of `env.launch`. I think with the introduction of process specific files we should consider adding a bullet point for `build` and one for `launch`.